src/MSBuild.UnitTests/MSBuildServer_Tests.cs

```diff
@@ -176,7 +180,11 @@ public void BuildsWhileBuildIsRunningOnServer()
         {
             _env.SetEnvironmentVariable("MSBUILDUSESERVER", "1");
             TransientTestFile project = _env.CreateFile("testProject.proj", printPidContents);
-            TransientTestFile sleepProject = _env.CreateFile("napProject.proj", sleepingTaskContents);
+
+            string fileName = "marker.txt";
+            string? dir = Path.GetDirectoryName(project.Path);
+            string markerFilePath = Path.Combine(dir!, fileName);
```

Comment:
I would prefer to use `_env.ExpectFile()` here so that the marker gets cleaned up.

Reply:
Thank you! I learned more about `TestEnvironment` from this.